src/tracks/playabletrack/wavetrack/ui/SpectrumView.h

```diff
@@ -22,6 +68,7 @@ class SpectrumView final : public WaveTrackSubView
 
 public:
    using WaveTrackSubView::WaveTrackSubView;
+   SpectrumView(WaveTrackView &waveTrackView);
```

Comment:
One-argument constructors should usually be `explicit` unless you really want implied conversions.